tool_quality_checks/sql_background_views.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+# TODO aanlegjaar toevoegen        year AS begindatum,
+# TODO putmateriaal toevoegen      material AS materiaalput,
+
+sql_background_views = {

```

Comment:
Zou dit duidelijker zijn als een aparte directory met losse sql files? Als je ze `background_view_put.sql` en `background_view_leiding.sql` enzo noemt. kan je ze met een kleine `sql_background_view("put")` functie inlezen.

Voordelen kunnen zijn:

- Handige syntax highlighting in je editor.
- Je kan een automatische sql codechecker eroverheen gooien.
- Misschien voor jullie meer leesbaarheid.

Maar dan moet het qua sql syntax wel goed gaan met van die dingen als `{schema}` :-)

Enfin, geen idee of jullie het makkelijk vinden. Ik gooi het maar in de groep.